solutions/bio1/OnTheRightTrack/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { SCodeBlock, SSubtitle, SText, STitle, SCode, } from "components";
+
+export const OnTheRightTrack = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        This is a quite straightforward implementation problem, and with the small-ish n $(n {'<'}= 10,000)$, a straightforward simulation is in order.
+      </SText>
+
+      <STitle>Solution</STitle>
+      <SText>
+        With larger problems (often Question $2$), it is useful to define a set of classes, and some methods embedded in them to help solve the problem.
+        In this problem, we can define a class, <SCode>Stop</SCode>, which will store the type of stop (lazy or flip flop) it is, which stops it is connected to, and which curved track the next train going through the straight one will exit through.
+        We will also update that specific curved track as each train passes through.
+        With this <SCode>Stop</SCode> class, tracking the train's movement is easy, and we can do this using a map which will store each stop indexed by it's character($A, B$, etc).
+        A helpful tip, when you use a map with a custom struct/class, be sure to implement a constructor which takes no parameters (which can be empty).
```

Comment:
The first comma should be a colon instead. Also, mention that this is a C++-specific tip.